sdk/monitor/azure-opentelemetry-exporter-azuremonitor/pom.xml

```diff
@@ -34,7 +34,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.12.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.13.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
What was added that made this change to an unreleased dependency? If this is remaining an unreleased dependency you'll need to update the version tag

Reply:
Had to take a dependency on unreleased `azure-core-tracing-opentelemetry` which in turn depends on unreleased core (current).